website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -8,7 +8,7 @@ dbt Core v1.3 included first-ever support for Python models. Note that only [spe
 
 We encourage you to:
 - Read [the original discussion](https://github.com/dbt-labs/dbt-core/discussions/5261) that proposed this feature.
-- Contribute to [best practices for developing Python models in dbt](https://github.com/dbt-labs/docs.getdbt.com/discussions/1811).
+- Contribute to [best practices for developing Python models in dbt](https://discourse.getdbt.com/c/discussions/21).
```

Comment:
We'll want a link to an actual discussion not just the category - @dbeatty10 can help get this set up.

Reply:
created https://discourse.getdbt.com/t/dbt-python-model-dbt-py-best-practices/5204, updating the PR